ddtrace/contrib/internal/logging/patch.py

```diff
@@ -93,6 +96,9 @@ def _w_makeRecord(func, instance, args, kwargs):
 
 
 def _w_StrFormatStyle_format(func, instance, args, kwargs):
+    if not config._logs_injection:
+        # log injection is opt-in for structured logging
```

Comment:
On by default means it should be opt-out, right?

Reply:
yup typo